frontend/helpers/cache-control-helpers.ts

```diff
@@ -16,8 +16,14 @@ export function hasDisableCacheGets(context: GetServerSidePropsContext) {
       return true;
    }
 
-   const disableCacheGets = context.query.disableCacheGets !== undefined;
-   return disableCacheGets;
+   const { disableCacheGets, revisionid } = context.query;
+
+   if (disableCacheGets !== undefined) {
+      return true;
+   }
+
+   const wantsHeadRevision = revisionid?.toString().toUpperCase() === 'HEAD';
```

Comment:
If we can do it, I think it would be better to _not_ centralize this logic and instead add a "hook" when setting up caching in the controller. We don't want to add this behavior to every route that sets up caching.

Reply:
I wasn't sure if opt-in or always made the most sense. Easy enough to change.